src/components/calcite-block/calcite-block.tsx

```diff
@@ -1,5 +1,5 @@
 import { Component, Element, Event, EventEmitter, Host, Prop, h, VNode } from "@stencil/core";
-import { CSS, SLOTS, TEXT, HEADING_LEVEL } from "./resources";
+import { CSS, SLOTS, TEXT, HEADING_LEVEL, TOGGLE_ICON } from "./resources";
```

Comment:
nitpick: We usually call this const `ICONS`.

Reply:
Thanks @driskull. Updated.